checkers/forbidden_import_call_checker.py

```diff
@@ -0,0 +1,36 @@
+"""checker for detecting call __import__.
+"""
+
+import astroid
+from pylint.interfaces import IAstroidChecker
+from pylint.checkers import BaseChecker
+from pylint.checkers.utils import check_messages
+
+
+class ImportFunctionChecker(BaseChecker):
+
+    __implements__ = IAstroidChecker
+
+    name = 'forbidden_import_call'
+    msgs = {'E9995': ('Call the function __import__ on %s',
+                      'func__import__not_allowed',
+                      'Used when you  call the function __import__ to import a '
+                      'module'),
+           }
+    # this is important so that your checker is executed before others
+    priority = -1
+
+    @check_messages("func__import__not_allowed")
+    def visit_call(self, node):
```

Comment:
@Jasminewwwu I've given this one a little thought. Rather than create a new class, I would like you to add this method to the existing import checker class. You can use the same message as the existing checker, as well.